CalibTracker/SiStripCommon/test/testProduceCalibrationTree_cfg.py

```diff
@@ -94,13 +95,22 @@
 ## process.L1T1.L1TechTriggerSeeding = cms.bool(True)
 ## process.L1T1.L1SeedsLogicalExpression = cms.string('(40 OR 41) AND NOT (36 OR 37 OR 38 OR 39)')
 
-process.EventInfo = cms.EDAnalyzer("ShallowTree",
-                                   outputCommands = cms.untracked.vstring('drop *',
+compressionSettings = 201
+process.EventInfo = cms.EDAnalyzer("ShallowTree", 
+					CompressionSettings = cms.untracked.int32(compressionSettings),
+                            		outputCommands = cms.untracked.vstring('drop *',
                                                                           'keep *_shallowEventRun_*_*',
                                                                           )
                                    )
+process.gainCalibrationTreeStdBunch.CompressionSettings = cms.untracked.int32(compressionSettings)
+process.gainCalibrationTreeStdBunch0T.CompressionSettings = cms.untracked.int32(compressionSettings)
+process.gainCalibrationTreeIsoMuon.CompressionSettings = cms.untracked.int32(compressionSettings)
+process.gainCalibrationTreeIsoMuon0T.CompressionSettings = cms.untracked.int32(compressionSettings)
+process.gainCalibrationTreeAagBunch.CompressionSettings = cms.untracked.int32(compressionSettings)
+process.gainCalibrationTreeAagBunch0T.CompressionSettings = cms.untracked.int32(compressionSettings)
```

Comment:
@echabert @mdelcourt, what about putting those upstream into:

https://github.com/cms-sw/cmssw/blob/master/CalibTracker/SiStripChannelGain/python/ntuple_cff.py#L44-L56

? 
Do you think there is actually any reason why we do *not* want to apply the further compression in any production?
This shall avoid doing it twice (here and in `MakeCalibrationTrees/produceCalibrationTree_template_cfg.py`.
Also I checked that the stuff that runs in PCL doesn't go through the `ntuple_cff` but redefines the shallow Trees from scratch:

https://github.com/cms-sw/cmssw/blob/master/Calibration/TkAlCaRecoProducers/python/ALCARECOPromptCalibProdSiStripGains_cff.py#L76-L82

so it would not be affected

Reply:
The motivation was to not slow down any production beyond the calibTree one. If ntuple_cff.py is not used elsewhere, then I agree that it is the best place to put changes.

Reply:
I also agree it seems to be best to add it upstream.